src/Compilers/CSharp/Portable/CommandLine/CommandLineParser.cs

```diff
@@ -1039,7 +1039,7 @@ public new CSharpCommandLineArguments Parse(IEnumerable<string> args, string bas
                 preprocessorSymbols: defines.ToImmutableAndFree(),
                 documentationMode: parseDocumentationComments ? DocumentationMode.Diagnose : DocumentationMode.None,
                 kind: SourceCodeKind.Regular
-            );
+            ).WithFeatures(features.ToImmutableDictionary(feature => feature, feature => "true"));
```

Comment:
Should that go into a separate PR? We're doing the same thing for CSharpCompilationOptions, and I would guess that both should be updated (or this one could grow). VB should get the same change too (but VB also needs the equivalent of this PR applied to its parse options).

https://github.com/khyperia/roslyn/blob/FeaturesFlag/src/Compilers/CSharp/Portable/CommandLine/CommandLineParser.cs#L1064